### PR TITLE
upgraded the project to ionic 3 and added lazy loading

### PR DESCRIPTION
**Pull request by kevinburnside**
Just needs reviews

### CODE REVIEW

File: src/pages/paymentInfo/paymentinfo.module.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+import { IonicPageModule } from 'ionic-angular';
+
+@NgModule({
+  declarations: [PaymentInfoPage],
+  imports: [IonicPageModule.forChild(PaymentInfoPage)],
+  entryComponents: [
+    PaymentInfoPage
+  ]
+})
+export class PaymentInfoPageModule { }
```
Comment (aagavin) [inline src/pages/paymentInfo/paymentinfo.module.ts]: This file also needs to be name `paymentInfo.module.ts`

**Review by aagavin — CHANGES_REQUESTED** [review]
build failing to naming issue.

**Comment by kevinburnside** [comment]
changed the paymentinfoPage error.

File: src/pages/paymentInfo/paymentinfo.module.ts (hunk 1 above)
Comment (kevinburnside) [reply src/pages/paymentInfo/paymentinfo.module.ts]: changed the paymentinfoPage error.

Comment (aagavin) [reply src/pages/paymentInfo/paymentinfo.module.ts]: @kevinburnside the file `paymentinfo.module.ts` still needs to be renamed to`paymentInfo.module.ts`

**Review by aagavin — CHANGES_REQUESTED** [review]
`paymentinfo.module.ts` still needs to be `paymentInfo.module.ts`

**Review by aagavin — APPROVED** [review]

**Pull request merged**